src/plugins/load.js

```diff
@@ -37,10 +37,11 @@ module.exports = function (Plugins) {
 	};
 
 	Plugins.prepareForBuild = function (callback) {
-		Plugins.cssFiles.length = 0;
-		Plugins.lessFiles.length = 0;
-		Plugins.clientScripts.length = 0;
-		Plugins.acpScripts.length = 0;
+		Plugins.cssFiles = [];
+		Plugins.lessFiles = [];
+		Plugins.clientScripts = [];
+		Plugins.acpScripts = [];
+		Plugins.soundpacks = [];
```

Comment:
Ya you're right I shouldn't have changed this I just ran into an issue where the array want defined or something and ended up changing all of them.